Core/Core/Common/CommonModels/AppEnvironment/FeatureFlags/GetEnvironmentFeatureFlags.swift

```diff
@@ -23,6 +23,7 @@ public enum EnvironmentFeatureFlags: String {
     case send_usage_metrics
     case mobile_offline_mode
     case account_survey_notifications
+    case restrict_student_access
```

Comment:
This is a development feature flag that affects all users. We need an account or course level flag to be used based on where the flag was put on API side.

Reply:
Hi @vargaat, thanks for the feedback. 

The `restrict_student_access` flag is set at the root account level, and I want to apply this to all users within that specific root account.

you meant to say that it would also apply for all root account users even if I enabled it to one. let me know if I misunderstood your comment. if yes can you please share some example feature flag that we already using in the code that I can use as a reference.